charts/datarepo-ui/templates/configmap.yaml

```diff
@@ -19,6 +19,7 @@ data:
     http {
       proxy_temp_path /tmp/proxy_temp;
       client_body_temp_path /tmp/client_temp;
+      include /etc/nginx/mime.types;
```

Comment:
I'm surprised it worked without this line, tbh!

Reply:
lol, the "how did this ever work!?" phase of debugging